bundler/lib/bundler/man/bundle-config.1.ronn

```diff
@@ -3,7 +3,13 @@ bundle-config(1) -- Set bundler configuration options
 
 ## SYNOPSIS
 
-`bundle config` [list|get|set|unset] [<name> [<value>]]
+`bundle config` list
+
+`bundle config`[ get] NAME
```

Comment:
```
`bundle config` [get] NAME
```
How about this?

Reply:
PR in #5856

Reply:
It was intentional, since if you omit the optionally part (`[ get]`), you're left with just a single space (`bundle config NAME`). I think I saw this style in man pages for other tools, and I like it, but if you prefer the other I don't really mind too much.

Reply:
I'm sure I got this from some man page, but the most common tools don't use it, so let's kill it.